plugin-cgroupv2/Cargo.toml

```diff
@@ -20,3 +20,8 @@ base64 = "0.22.1"
 
 [lints]
 workspace = true
+
+[profile.release]
+opt-level = 1
+lto = true
+codegen-units = 1
```

Comment:
Per se it's not a bad idea to enable more aggressive LTO, however:
- it should not be applied on a plugin (because it won't do anything)
- this modification should be in another PR
- the benefits should be demonstrated (how much do we gain in perf, how much do we loose in compilation time, what impact does it have on binary size?)